appveyor.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+set -e -v
+
+echo "C_COMPILER: $C_COMPILER"
+echo "D_COMPILER: $D_COMPILER"
+echo "D_VERSION: $D_VERSION"
+
+cd /c/projects/
+
+if [ $D_COMPILER == "dmd" ]; then
+    appveyor DownloadFile "http://downloads.dlang.org/releases/2.x/${D_VERSION}/dmd.${D_VERSION}.windows.7z" -FileName dmd2.7z
+    7z x dmd2.7z > /dev/null
+    export PATH=$PWD/dmd2/windows/bin/:$PATH
+    export DMD=/c/projects/dmd2/windows/bin/dmd.exe
+    dmd --version
+fi
+
+if [ $APPVEYOR_REPO_TAG == true ]; then
+    git clone --branch $APPVEYOR_REPO_TAG_NAME https://github.com/dlang/druntime
+    git clone --branch $APPVEYOR_REPO_TAG_NAME https://github.com/dlang/phobos
+else
+    git clone --branch master https://github.com/dlang/druntime
+    git clone --branch master https://github.com/dlang/phobos
+fi
+
```

Comment:
Not sure whether CircleCi supports it, but otherwise we need to manually merge the PR into master, see e.g.:

https://github.com/dlang/dmd/blob/d28274e1c666ce8c8905587b8d86945927302315/circleci.sh#L73-L82

Reply:
I think it does do the merge first, at least by the how it grabs the commit it looks like it does.